pop.js

```diff
@@ -0,0 +1,148 @@
+//  Javascript popup
+
+const projects = [
+  {
+    name: 'Proffesional Art<br>Printing Data',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    featuredImage: 'images/Snapshoot Portfolio.svg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.<br> It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    linkToLiveVersion: 'https://2004-okari.github.io/MY-PORTFOLIO/',
+    linkToSource: 'https://github.com/2004-okari/MY-PORTFOLIO',
```

Comment:
- [ ] Nice efforts so far. However, you still have to adjust the popup based on the changes asked by the previous reviewer.
- The background content is still not burred when the popup is clicked. The whole focus should be on the popup.
- The `buttons` are meant to be at the bottom. Proper use of grid can accomplish this to place the items 

### Your view:
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/50990925/1f8d7562-9810-4983-9a77-ddd8f1e9ad55)

### Figma:
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/50990925/a175eb56-e21b-46d0-8651-15f42d0ea057)